### PR TITLE
feat: Handle guest link (AR-2706) (AR-2708) (AR-2709) (AR-2707)

### PR DESCRIPTION
**Pull request by ohassine**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2706" title="AR-2706" target="_blank">StoryAR-2706</a>  Admin: create a guest link
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Description

This PR aims to handle guest room link : create, share and copy

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

Go to conversation details --> Guests

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

Uploading screen-20230220-182051.mp4…

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/3/) **failed**.

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/5/) **failed**.

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/6/) **failed**.

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/7/) **failed**.

**Comment by AndroidBob** [comment]
Build [8](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/8/) **failed**.

**Comment by AndroidBob** [comment]
Build [9](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/9/) **failed**.

**Comment by AndroidBob** [comment]
Build [10](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/10/) **failed**.

**Comment by AndroidBob** [comment]
Build [11](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/11/) **failed**.

**Comment by AndroidBob** [comment]
Build [12](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/12/) **failed**.

**Comment by AndroidBob** [comment]
Build [13](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/13/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.0(5795137)(5795137)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.0(5795137)(5795137)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt [hunk 1]
```diff
@@ -155,17 +151,16 @@ private fun SelfUserProfileContent(
...
                     .padding(internalPadding)
             ) {
                 LazyColumn(
                     modifier = Modifier
                         .weight(1F)
                         .fillMaxWidth()
-                        .fillMaxHeight()
-                        .scrollable(state = scrollState, orientation = Orientation.Vertical)
+//                        .fillMaxHeight()
+//                        .scrollable(state = scrollState, orientation = Orientation.Vertical)
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt]: Should this be removed?

File: app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt [hunk 2]
```diff
@@ -155,17 +151,16 @@ private fun SelfUserProfileContent(
             Column(
                 modifier = Modifier
                     .fillMaxWidth()
-                    .fillMaxHeight()
-                    .scrollable(state = scrollState, orientation = Orientation.Vertical)
+//                    .scrollable(state = scrollState, orientation = Orientation.Vertical)
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt]: Should this be removed?

**Review by alexandreferris — COMMENTED** [review]
Looking good! Just a quick question on commented code

File: app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt (hunk 1 above)
Comment (ohassine) [reply app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt]: no no, I commented that for testing purposes and I forgot to uncomment 😬

**Review by alexandreferris — APPROVED** [review]
👌🏻

**Comment by AndroidBob** [comment]
Build [14](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/14/) **failed**.

**Comment by AndroidBob** [comment]
Build [15](https://10.10.124.134/job/wire-android-reloaded/job/PR-1463/15/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.0(5800710)(5800710)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.0(5800710)(5800710)-staging-compat.apk)

**Pull request merged**